TypeCobol/Compiler/Scanner/Token.cs

```diff
@@ -295,8 +295,13 @@ public virtual string Text
         /// </summary>
         public LiteralTokenValue LiteralValue { get; set; }
 
+        /// <summary>
+        /// ScanState associated to this token if any, null otherwise.
+        /// </summary>
+        public MultilineScanState ScanStateSnapshot { get; set; }
```

Comment:
Better make it internal as it is used during parsing but not necessarily a result to be exposed to clients. Besides only PartialCobolWord tokens will get this property set so it is confusing for callers.